### PR TITLE
[BEAM-272][flink] remove dependency on Dataflow Runner

### PR DESCRIPTION
**Pull request by mxm**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [X] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [X] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [X] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [X] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

File: runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java [hunk 1]
```diff
@@ -52,7 +56,7 @@
    */
   @Description("Dataflow job name, to uniquely identify active jobs. "
```
Comment (swegner) [inline runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java]: Should this be "Flink job name"?

**Comment by aljoscha** [comment]
+1 seems good except for the comment by @swegner

File: runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java (hunk 1 above)
Comment (mxm) [inline runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java]: Yep, another relic. Thanks!

**Comment by mxm** [comment]
Thanks. Merging.

**Pull request merged**